tests/schema/tests.py

```diff
@@ -5298,3 +5303,29 @@ def test_ci_cs_db_collation(self):
             if connection.vendor == "postgresql":
                 with connection.cursor() as cursor:
                     cursor.execute("DROP COLLATION IF EXISTS case_insensitive")
+
+
+@isolate_apps("schema")
+@skipUnlessAnyDBFeature("supports_json_field", "supports_stored_generated_columns")
```

Comment:
Both must be supported
```suggestion
@skipUnlessDBFeature("supports_json_field", "supports_stored_generated_columns")
```

Reply:
Fixed